app/src/main/java/com/kwartracker/android/transactions/ui/details/DetailsTransactionFragment.kt

```diff
@@ -4,14 +4,19 @@ import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import android.widget.Toast
 import androidx.databinding.DataBindingUtil
 import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
 import com.kwartracker.android.R
 import com.kwartracker.android.databinding.FragmentDetailsTransactionBinding
 import com.kwartracker.android.transactions.ui.main.TransactionFragment
+import com.kwartracker.android.transactions.ui.main.TransactionFragmentDirections
+import com.kwartracker.android.widgets.ConfirmationDialog
 
 class DetailsTransactionFragment : Fragment() {
     lateinit var binding: FragmentDetailsTransactionBinding
+    private lateinit var confirmationDialog: ConfirmationDialog
```

Comment:
@danacealfonso 
This property is initialized without being used.

Reply:
Removed in c35538db83728f800ede18672975ef2b1b4d22c1